File: Beginner/Uri1012.java
package Beginner;

import java.util.Locale;
import java.util.Scanner;

public class Uri1012 {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		double a = sc.nextDouble();
		double b = sc.nextDouble();
		double c = sc.nextDouble();
		
		double triangle = (a * c)/2;
		double circle = 3.14159 * (Math.pow(c, 2));
		double trapezium = ((a + b) * c)/2;
		double square = Math.pow(b, 2);
		double rectangle = a * b;
		
		System.out.printf("TRIANGULO: %.3f\n", triangle);
		System.out.printf("CIRCULO: %.3f\n", circle);
		System.out.printf("TRAPEZIO: %.3f\n", trapezium);
		System.out.printf("QUADRADO: %.3f\n", square);
		System.out.printf("RETANGULO: %.3f\n", rectangle);
	}

}

File: src/beginner/Uri1012.py
linha1 = input().split(" ")
A, B, C = linha1

triangulo = (float(A) * float(C)) / 2
circulo = 3.14159 * float(C) ** 2
trapezio = (float(C) * (float(A) + float(B))) / 2
quadrado = float(B) ** 2
retangulo = float(A) * float(B)

print("TRIANGULO: %0.3f" %triangulo)
print("CIRCULO: %0.3f" %circulo)
print("TRAPEZIO: %0.3f" %trapezio)
print("QUADRADO: %0.3f" %quadrado)
print("RETANGULO: %0.3f" %retangulo)